lsp4e.aggrcon

```diff
@@ -1,19 +1,20 @@
 <?xml version="1.0" encoding="ASCII"?>
 <aggregator:Contribution xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:aggregator="http://www.eclipse.org/cbi/p2repo/2011/aggregator/1.1.0" label="LSP4E">
-  <repositories location="https://download.eclipse.org/lsp4e/releases/0.26.2/" description="LSP4E">
-    <features name="org.eclipse.lsp4e" versionRange="[0.18.10.202405280703]">
+  <repositories location="https://download.eclipse.org/lsp4e/releases/0.26.4/" description="LSP4E">
+    <features name="org.eclipse.lsp4e" versionRange="[0.18.12.202408150719]">
       <categories href="simrel.aggr#//@customCategories[identifier='General%20Purpose%20Tools']"/>
     </features>
-    <features name="org.eclipse.lsp4e.debug" versionRange="[0.15.8.202405212039]">
+    <features name="org.eclipse.lsp4e.debug" versionRange="[0.15.10.202408140718]">
       <categories href="simrel.aggr#//@customCategories[identifier='General%20Purpose%20Tools']"/>
     </features>
     <features name="org.eclipse.lsp4e.jdt" versionRange="[0.13.0.202402070603]">
       <categories href="simrel.aggr#//@customCategories[identifier='General%20Purpose%20Tools']"/>
     </features>
-    <features name="org.eclipse.lsp4e.source" versionRange="[0.18.10.202405280703]"/>
-    <features name="org.eclipse.lsp4e.debug.source" versionRange="[0.15.8.202405212039]"/>
+    <features name="org.eclipse.lsp4e.source" versionRange="[0.18.12.202408150719]"/>
+    <features name="org.eclipse.lsp4e.debug.source" versionRange="[0.15.10.202408140718]"/>
     <features name="org.eclipse.lsp4e.jdt.source" versionRange="[0.13.0.202402070603]"/>
   </repositories>
   <contacts href="simrel.aggr#//@contacts[email='mistria@redhat.com']"/>
   <contacts href="simrel.aggr#//@contacts[email='jonah@kichwacoders.com']"/>
+  <contacts href="simrel.aggr#//@contacts[email='pcr@avaloq.com']"/>
```

Comment:
This is a reference to a contact in the main resource so you can’t just edit it textually and would need to commit the other resource. 

I can help further when I’m back in the office.

Reply:
Thanks for the help. I think I have figured it out and the new version of the PR should build fine. If not, I might need help 👍

Reply:
The looks better. Nicer would have been to amend and force push. But let’s not get too fussy.